server/service/individualServices/ReadVlanInterfaceData.js

```diff
@@ -1,28 +1,340 @@
+      }
+      /****************************************************************************************************
+      * process EthernetContainerStatus for each air_interface
+      *   RequestForProvidingAcceptanceDataCausesDeterminingTheWanPortRole.EthernetContainerStatus
+      *     MWDI://core-model-1-4:network-control-domain=cache/control-construct={mount-name}
+      *       /logical-termination-point={uuid}/layer-protocol={local-id}
+      *         /ethernet-container-2-0:ethernet-container-pac/ethernet-container-status?fields=interface-status
+      *****************************************************************************************************/
+      let ethernetContainerStatusResponse = await fetchServingEthernetContainerStatus(mountName, requestHeaders, traceIndicatorIncrementer, clientContainerLtp, clientAndFieldParamsForEthernetContainerStatus);
+      if (Object.keys(ethernetContainerStatusResponse).length != 0) {
+        if (ethernetContainerStatusResponse.servingEthernetContainerStatus != undefined) {
+          configuredWanPortRole.servingEthernetContainerStatus = ethernetContainerStatusResponse.servingEthernetContainerStatus;
+        }
+        traceIndicatorIncrementer = ethernetContainerStatusResponse.traceIndicatorIncrementer;
+      } else {
+        console.log(`${ethernetContainerStatusCallback} for ${airInterfaceUuid} is not success`);
+      }
+      configuredWanPortRoleList.push(configuredWanPortRole);
+    }
+    configuredWanPortRoleListResponse = {
+      configuredWanPortRoleList: configuredWanPortRoleList,
+      traceIndicatorIncrementer: traceIndicatorIncrementer
+    };
+    return configuredWanPortRoleListResponse;
+  } catch (error) {
+    console.log(error);
+    return new createHttpError.InternalServerError();
+  }
+}
+
+/**
+ * Fetches original-ltp-name for ethernet-container ltp associated to chosen wire/air layer
+ * @param {String} mountName Identifier of the device at the Controller
+ * @param {Object} ltpStructure ControlConstruct provided from cache
+ * @param {Object} requestHeaders Holds information of the requestHeaders like Xcorrelator , CustomerJourney,User etc.
+ * @param {Integer} traceIndicatorIncrementer traceIndicatorIncrementer to increment the trace indicator
+ * @param {Object} ltp chosen ltp instance of wire/air layer
+ * @param {Object} clientAndFieldParams Holds information of the query and field parameters to use in request url
+ * @returns {Object} returns original-ltp-name value and traceIndicatorIncrementer
+ */
+async function fetchOriginalLtpName(mountName, ltpStructure, requestHeaders, traceIndicatorIncrementer, ltp, clientAndFieldParams) {
+  let originalLtpNameResponse = {};
+  let pathParamList = [];
+  let interfaceListForOriginalLtpName = [PURE_ETHERNET_STRUCTURE.MODULE + PURE_ETHERNET_STRUCTURE.LAYER_PROTOCOL_NAME, ETHERNET_CONTAINER.MODULE + ETHERNET_CONTAINER.LAYER_PROTOCOL_NAME];
```

Comment:
For AirInterface, the client shall be PureEthernetStructure or HybridMicrowaveStructure. Kindly please update the code to support in both the cases. Thanks.